src/main/java/seedu/address/logic/commands/AddPropertyCommand.java

```diff
@@ -61,8 +63,16 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
         Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = new Person(personToEdit.getName(), personToEdit.getPhone(),
-                personToEdit.getEmail(), personToEdit.getTags(), personToEdit.getAppointment(), property);
+        Person editedPerson;
+        if (personToEdit instanceof Buyer) {
+            editedPerson = new Buyer(personToEdit.getName(), personToEdit.getPhone(), personToEdit.getEmail(),
```

Comment:
Abstract common inputs to the Buyer and Seller, such as .getName(), .getPhone() ...